packages/components/addon/components/hds/code-block/index.hbs

```diff
@@ -0,0 +1,30 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<div class="hds-code-block-wrapper">
```

Comment:
should we use `<pre><code>` here, as I've seen done in some implementations?

Reply:
CodeMirror generates a bunch of `div`s with `<pre>` tags nested inside for each line of code.

The `code` tag is only supposed to be used to wrap phrasing content, not block-level content such as a `div`.

We could look at this more closely when we implement the actual component however.